src/components/molecules/UserProfilePicture/UserProfilePicture.types.ts

```diff
@@ -0,0 +1,12 @@
+import { User } from "types/User";
```

Comment:
This file was removed in a previous PR, now it seems to be back? I expect this may be because of something to do with the wrong branch being merged in/etc, as mentioned in another comment.

Reply:
Well I merged with origin/master

Reply:
*nods* I figured you may have. Our default branch is staging, so we never merge master into PR’s like this.